pkg/machine/qemu/options_windows_amd64.go

```diff
@@ -5,11 +5,10 @@ var (
 )
 
 func (v *MachineVM) addArchOptions() []string {
-	// "max" level is used, because "host" is not supported with "whpx" acceleration
-	// "vmx=off" disabled nested virtualization (not needed for podman)
-	// QEMU issue to track nested virtualization: https://gitlab.com/qemu-project/qemu/-/issues/628
-	// "monitor=off" needed to support hosts, which have mwait calls disabled in BIOS/UEFI
-	opts := []string{"-machine", "q35,accel=whpx:tcg", "-cpu", "max,vmx=off,monitor=off"}
+	// "qemu64" level is used, because "host" is not supported with "whpx" acceleration.
+	// It has performance penalty, but is a stable choice for running on bare metal
```

Comment:
are you saying that qemu64 has a performance penalty ?

Reply:
Removed the part about performance penalty and kept only motivational part as future code contributors are believed to understand QEMU settings differences of different emulation levels. End users, which will be tinkering with the settings should understand the settings as well.